Makefile

```
@@ -10,4 +10,11 @@ linux:
 	GOOS=linux GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-linux-$(VERSION) .
 
 windows:
-	GOOS=windows GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-win-$(VERSION) .
\ No newline at end of file
+	GOOS=windows GOARCH=amd64 go build -ldflags="$(FLAGS)" -o bin/ecs-upload-task-win-$(VERSION) .
+
+release-multi-arch:
+	mkdir -p bin
+	gox -os="linux darwin windows" -arch="amd64 arm64" -output="./bin/{{.Dir}}_{{.OS}}_{{.Arch}}"      
+
+clean:
+	rm -f bin/*
```

You could do this too:
```suggestion
	rm -rf bin
	
```